eslint.config.js

```diff
@@ -1,44 +1,41 @@
-import { fixupConfigRules } from '@eslint/compat';
-import pluginJs from '@eslint/js';
-import * as reactQuery from '@tanstack/eslint-plugin-query';
-import pluginImport from 'eslint-plugin-import';
```

Comment:
I belive you accidentally delete pluginImport

Reply:
Yes. I focues more on `eslint.cjs` instead of previous `eslint.config.js` state. I will fix this.